extensions/common/vault/vault-gcp/README.md

```diff
@@ -0,0 +1,14 @@
+# GCP Secret Manager Vault
+
+The vault-gcp extension is an implementation of the Vault interface based on GCP Secret Manager.
+Arbitrary key names are possible through the key sanitation feature. 
+
+## Settingss
```

Comment:
typo. Also, this is not needed, since all settings are annotated with `@Setting` and get picked up by the `autodoc` annotation processor.

Reply:
Removed, thank you!